tests/Doctrine/Tests/ORM/Functional/Ticket/GH10049/SetInheritedReadOnlyPropertyValueTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\ORM\Functional\Ticket\GH10049;
+
+use Doctrine\Tests\OrmFunctionalTestCase;
+
+/**
+ * @requires PHP 8.1
+ * @doesNotPerformAssertions
```

Comment:
I think this needs to be moved before the `test` method

Reply:
…damn :see_no_evil: